### ISSUE TITLE
License doesn't allow redistribution

### ISSUE THREAD

**Issue by xhochy** [conda-forge/moleculekit-feedstock#73]
Being made aware though the Matrix chat, this software doesn't allow open redistribution and thus we are not allowed to actually host it on conda-forge: https://github.com/Acellera/moleculekit/blob/ba2e8c6cba39ed8f6aa097b9452c90fa41d44fb2/LICENSE#L10

cc @conda-forge/moleculekit @conda-forge/core

**Comment by stefdoerr** [comment]
Yes, you are right about this. I can discuss this a bit at Acellera if we are willing to relax that condition in the license.

**Comment by ocefpaf** [comment]
We should remove it until then.

**Comment by beckermr** [comment]
I am also going to archive the feedstock.
